### PR TITLE
[REF] remove use of moment as it is deprecated

### PR DESCRIPTION
**Pull request by ged-odoo**

### CODE REVIEW

File: addons/mail/static/src/new/utils.js [hunk 1]
```diff
@@ -291,3 +291,11 @@ export function useScrollSnapshot(refName, { onWillPatch: p_onWillPatch, onPatch
         });
     });
 }
+
+const { DateTime } = luxon;
```
Comment (seb-odoo) [inline addons/mail/static/src/new/utils.js:295]: Shouldn't global imports typically be at the top of the file?

**Review by seb-odoo — APPROVED** [review]

File: addons/mail/static/src/new/utils.js [hunk 2]
```diff
@@ -291,3 +291,11 @@ export function useScrollSnapshot(refName, { onWillPatch: p_onWillPatch, onPatch
...
+
+export function computeDelay(dateStr) {
```
Comment (phenix-factory) [inline addons/mail/static/src/new/utils.js:297]: Would name this `computeDelayFromNow` that use a `computeDelay` function that accept 2 dateStr

File: addons/mail/static/src/new/utils.js (hunk 1 above)
Comment (ged-odoo) [reply addons/mail/static/src/new/utils.js:295]: technically, this is not an import :). But at some point, I think we should reorganize the utility functions in a better way: `utils/array`, `utils/hooks`, `utils/dates` and so on.

File: addons/mail/static/src/new/utils.js (hunk 2 above)
Comment (ged-odoo) [reply addons/mail/static/src/new/utils.js:297]: sure, once we need it. I think the idea is to merge PRs and adjust, so we don't have to manage many prs in the meantime until it's perfect (it never is anyway)

**Pull request merged**